infra/aws/terraform/registry-sandbox-k8s-io-image-layers/main.tf

```diff
@@ -14,23 +14,32 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/*
-This file defines:
-- Required provider versions
-- Storage backend details
-- alias for the provider per region
-*/
-
 terraform {
-  backend "gcs" {
-    bucket = "k8s-infra-tf-aws"
-    prefix = "oci-proxy"
-  }
+  required_version = "~> 1.1"
```

Comment:
I think we should use a newer Terraform version.

Reply:
I verified that using `~> 1.1` allows using newer Terraform versions (e.g. 1.5) so this can stay IMO